notebooks/analyze_looping_across_lending_protocols.ipynb

```diff
@@ -807,7 +695,7 @@
     "import matplotlib.ticker as ticker\n",
     "\n",
     "# Function to calculate total amounts of debt for each token\n",
-    "def calculate_total_debt(user_actions):\n",
+    "def calculate_total_debt(user_actions: DefaultDict[str, List[Tuple[str, str, float, str]]]) -> Tuple[DefaultDict[str, int], DefaultDict[str, int]]:\n",
     "    total_debt = defaultdict(float)\n",
     "\n",
```

Comment:
[Not relevant for this particular line] When computing per-user stats in the charts at the bottom of the ntb, use the number of active users, not all users. This is because zkLend has the most users, but most of them are inactive.